hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/clients/AllTopicClientsService.java

```diff
@@ -0,0 +1,10 @@
+package pl.allegro.tech.hermes.management.domain.clients;
+
+import pl.allegro.tech.hermes.api.TopicName;
+
+import java.util.List;
+
+public interface AllTopicClientsService {
```

Comment:
[not so important] Do we need this interface if its has only one implementation ? Then you will not need to create names like `AllTopicClientsServiceImplementation`

Reply:
I would like to change the implementation in our internal infrastructure, so it would be nice to have this interface

Reply:
I guess the interface is here to inject an internal implementation (for example to return emails fetched from an internal system instead of owner ids). However, I agree that `AllTopicClientsServiceImplementation` could be named more descriptively - in my opinion something like `DefaultAllTopicClientsService` or `OwnerIdBasedAllTopicClientsService` would be better.

Reply:
That's correct, thanks. I renamed `AllTopicClientsServiceImplementation` to `DefaultAllTopicClientsService`.